src/components/TextFieldChip/TextFieldChips.tsx

```diff
@@ -190,6 +167,51 @@ const TextFieldChips = React.forwardRef(
       })
     }
 
+    const handleClickAway = () => {
+      if (!isFocusingRef.current) {
+        return
+      }
+      if (chipIndexEditable !== null) {
+        clearChipIndexEditable()
+        clearInputValue()
+      } else if (addOnBlur) {
+        const inputValueTrimed = inputValue.trim()
+        if (inputValue.length > 0) {
```

Comment:
```inputValueTrimed.length > 0``` ?

Reply:
Ok you copied the code from `handleKeyDown`, but the feature here is only adding a chip, or update one on blur, that's all.

Reply:
If the user clicks away while the value is " ", this clears the input field.

Would you prefer any spaces to be left in place?